src/plugins/plugin.tooltip.js

```diff
@@ -1023,8 +1023,14 @@ export class Tooltip extends Element {
 			}
 		}
 
+		// When there are multiple items shown, but the tooltip position is nearest mode
+		// an update may need to be made because our position may have changed even though
+		// the items are the same as before.
+		const position = positioners[options.position].call(me, active, e);
+		const positionChanged = this.caretX !== position.x || this.caretY !== position.y;
```

Comment:
I don't quite get this logic. Won't this line be `true` as long as the cursor isn't positioned in the same exact place as the tooltip? It seems like it will pretty much always be `true` if I understand correctly

Reply:
Its the tooltip position that needs to change.

Reply:
Yup, it's the tooltip position. It happens when the tooltip is in "nearest" position mode and so the mouse position matters more.

Reply:
I think my confusion was due to not understanding the names `caretX` and `caretY`. I guess caret means the little triangle on the tooltip, but I thought it was referring to the mouse. So I thought it was saying if the tooltip isn't where the mouse is. But I guess this line is just saying if the new tooltip location isn't the same as the old tooltip location, is that right?